File: python/pad_array.py
def pad(array, min_size, value = None):
    return array if value == 0 else padding(array, min_size, value)


def padding(array, min_zise, value):
    start_at = len(array)
    for i in range(start_at, min_zise):
        array.append(value)
    return array

File: js/padArray.js
export const pad = (array, minSize, value=null) => {
    return minSize <= array.length ? array: padding(array, minSize, value)
}

function padding(array, minSize, value) {
    let startAt = array.length
    for (let i = startAt; i < minSize; i++) {
        value == null ? array[i] = null : array[i] = value
    }
    return array
}


